lib/tasks/dev.rake

```diff
@@ -3,6 +3,33 @@ if Rails.env.development? || Rails.env.test?
   require "factory_bot"
 
   namespace :pomegranate do
+    desc 'Reindex a resource from a IIIF Manifest URL within an Exhibit'
+    task :reindex, %i[manifest exhibit] => [:environment] do |_t, args|
+      manifest = args[:manifest]
+      exhibit_slug = args[:exhibit]
+      exhibit = Spotlight::Exhibit.find_by(slug: exhibit_slug)
+      iiif_resource = IIIFResource.find_or_initialize_by(url: manifest, exhibit: exhibit)
+      iiif_resource.save_and_index_now
+      puts "Reindexed the document for #{manifest}"
+    end
+
+    desc 'Clear the SolrDocumentSidecar reference fields'
+    task :sidecar_clean_references, [:exhibit] => [:environment] do |_t, args|
+      exhibit_slug = args[:exhibit]
+      exhibit = Spotlight::Exhibit.find_by(slug: exhibit_slug)
+      sidecars = Spotlight::SolrDocumentSidecar.where(exhibit: exhibit)
+      sidecars.each do |sidecar|
+        sidecar.data
+        valid_data = sidecar.data.reject do |k, v|
+          k.include?('references') && v.include?('iiif_manifest_paths')
+        end
+        next unless sidecar.data != valid_data
+        sidecar.data = valid_data
+        sidecar.save
+        puts "Updated the SolrDocumentSidecar for #{sidecar.document_id}"
```

Comment:
You'll probably want to reindex these items too. Maybe via `sidecar.resource.reindex_later`?